build.sh

```diff
@@ -64,11 +65,11 @@ else
     # No .git directory - That's a normal install.
     git clone https://github.com/HyNetwork/hysteria.git "$TMPDIR/hysteria"
     cd hysteria || exit 1
-    git checkout --force --quiet da16c88
+    git checkout --force --quiet 160145f7
     cd ..
     git clone https://github.com/v2fly/v2ray-core.git "$TMPDIR/v2ray-core"
     cd v2ray-core || exit 1
-    git checkout --force --quiet b4069f74
+    git checkout --force --quiet 9b526285
```

Comment:
i see the hysteria submodule update below but no v2ray submodule update

Reply:
I see both :) (I did forget to push up the v2ray-core change at first, so maybe you got in before that)

Reply:
now i see v2ray but no hysteria, but you said you were reverting hysteria so i guess that makes sense?